document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -643,6 +698,17 @@ private String prepareCast(String field, Object value) {
     }
   }
 
+  private String getInsertSQL() {
+    return String.format(
+        "INSERT INTO %s (%s,%s) VALUES( ?, ? :: jsonb)",
+        collectionName, ID, DOCUMENT, ID, DOCUMENT);
+  }
+
+  private String getUpdateSQL() {
```

Comment:
I think this can perfectly be static but I also wonder if we need to run the format every time, can't we format the SQL statement on construct and reuse it?

Reply:
It uses `collectionName` (table) and its per collection object - `PostgresCollection`. So, we can optimize a bit. I will raise a new refactor PR for all the similar SQL method statements.